packages/@okta/i18n/src/properties/login.properties

```diff
@@ -649,5 +649,8 @@ consent.required.privacyPolicy = Privacy Policy
 consent.required.consentButton = Allow Access
 consent.required.cancelButton = Don't Allow
 
+# Polling state
+polling.title = There are too many users trying to sign in right now. We will automatically retry in {0} seconds.
```

Comment:
@taylorlaubach-okta should we retake a look on this statement. Is it really reflecting what the situation is ? also we may like to shorten this message?